### PR TITLE
Add kubectl to local-up script

### PR DESCRIPTION
**Pull request by thockin**

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -64,7 +64,13 @@ do
...
 
 if [ "x$GO_OUT" == "x" ]; then
-    "${KUBE_ROOT}/hack/build-go.sh" cmd/kube-proxy cmd/kube-apiserver cmd/kube-controller-manager cmd/kubelet plugin/cmd/kube-scheduler
+    "${KUBE_ROOT}/hack/build-go.sh" \
+        cmd/kube-proxy \
+        cmd/kube-apiserver \
+        cmd/kube-controller-manager \
+        cmd/kubelet \
+        plugin/cmd/kube-scheduler \
+        cmd/kubectl
```
Comment (wojtek-t) [inline hack/local-up-cluster.sh]: nit: can you please sort them alphabetically?

Comment (thockin) [inline hack/local-up-cluster.sh]: sorted and pushed

**Comment by wojtek-t** [comment]
LGTM

@k8s-bot e2e test this please

**Comment by wojtek-t** [comment]
@k8s-bot e2e test this please

**Comment by wojtek-t** [comment]
@k8s-bot e2e test this please

**Pull request merged**